packages/core/src/api/clipboard/fromClipboard/acceptedMIMETypes.ts

```diff
@@ -1,6 +1,7 @@
 export const acceptedMIMETypes = [
   "vscode-editor-data",
   "blocknote/html",
+  "text/markdown",
```

Comment:
Out of curiosity, do you know any apps that use `text/markdown` for the clipboard? Since AFAIK it's not an official MIME type. I know Notion for example just puts Markdown in `text/plain`

Reply:
It's got an RFC: https://www.rfc-editor.org/rfc/rfc7763

It is trivial for us to implement, and makes it explicit rather than having to parse plain text.

I don't have example apps that use this. And, it'd be kind of annoying to check haha